From 27f60f11f6c3d048e42692476d949e35a90b863f Mon Sep 17 00:00:00 2001
From: Graham Leggett <minfrin@apache.org>
Date: Tue, 18 Sep 2018 21:45:59 +0000
Subject: [PATCH 171/504] mod_proxy_hcheck: Fix issues with interval
 determination. PR 62318 trunk patch: http://svn.apache.org/r1838937 2.4.x
 patch: svn merge -c 1838937 ^/httpd/httpd/trunk . +1: jim, ylavic, minfrin

git-svn-id: https://svn.apache.org/repos/asf/httpd/httpd/branches/2.4.x@1841266 13f79535-47bb-0310-9956-ffa450edef68
---
 CHANGES                          | 3 +++
 STATUS                           | 5 -----
 modules/proxy/mod_proxy_hcheck.c | 2 +-
 3 files changed, 4 insertions(+), 6 deletions(-)

diff --git a/modules/proxy/mod_proxy_hcheck.c b/modules/proxy/mod_proxy_hcheck.c
index bf557b71e9..2783a58e78 100644
--- a/modules/proxy/mod_proxy_hcheck.c
+++ b/modules/proxy/mod_proxy_hcheck.c
@@ -830,6 +830,7 @@ static void * APR_THREAD_FUNC hc_check(apr_thread_t *thread, void *b)
                  "%sHealth checking %s", (thread ? "Threaded " : ""),
                  worker->s->name);
 
+    worker->s->updated = now;
     if (hc->s->method == TCP) {
         rv = hc_check_tcp(baton);
     }
@@ -870,7 +871,6 @@ static void * APR_THREAD_FUNC hc_check(apr_thread_t *thread, void *b)
             }
         }
     }
-    worker->s->updated = now;
     apr_pool_destroy(baton->ptemp);
     return NULL;
 }
-- 
2.19.1

